Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 // Builds a module using https://github.com/jenkins-infra/pipeline-library
-buildPlugin(
-        configurations: [
-                [platform: 'linux', jdk: '11']
-        ])
+buildPlugin(useContainerAgent: true, configurations: [
+  [platform: 'linux', jdk: 17],
```

Comment:
```suggestion
  [platform: 'linux', jdk: 17],
  [platform: 'linux', jdk: 11],
```

Reply:
I'd preferred to not remove Linux Java 11 build and test

Reply:
Inconsistent with the archetype, but OK